x-pack/plugins/infra/public/containers/logs/log_position/log_position_state.ts

```diff
@@ -19,24 +25,35 @@ interface VisiblePositions {
 }
 
 export interface LogPositionStateParams {
+  initialized: boolean;
   targetPosition: TimeKeyOrNull;
-  isAutoReloading: boolean;
+  isStreaming: boolean;
+  liveStreamingInterval: number;
   firstVisiblePosition: TimeKeyOrNull;
   pagesBeforeStart: number;
   pagesAfterEnd: number;
   visibleMidpoint: TimeKeyOrNull;
   visibleMidpointTime: number | null;
   visibleTimeInterval: { start: number; end: number } | null;
+  startDate: string;
```

Comment:
Two questions:

- Can we somehow disambiguate the two intervals via their names?
- Could we stick to either a number or string representation for the dates? What would be the benefit of mixing them?

Reply:
Discussed in slack, but yes.

- `startDate` and `endDate` will be renamed to `startDateExpression` and `endDateExpression`. They represent the datemath expression of the datepicke. This name makes clearer what they are.

- `startTimestamp` and `endTimestamp` represent the parsed value of the expressions and will stay as `number`s.